src/components/BackButton.spec.js

```diff
@@ -0,0 +1,10 @@
+import {render, screen} from '@testing-library/react'
+  import BackButton from './BackButton'
+    
+  describe('BackButton', () => {
+      it('renders', () => {
```

Comment:
you could test the onClick function

Reply:
Tests and Storybook are coming later ;-)